dlp/snippets/deid_test.py

```diff
@@ -39,6 +39,14 @@
 DATE_SHIFTED_AMOUNT = 30
 DATE_FIELDS = ["birth_date", "register_date"]
 CSV_CONTEXT_FIELD = "name"
+TABLE_DATA = {
```

Comment:
Same declaration is present in another PR as well. Needs resolving conflicts once previous PR is merged.

Reply:
yes!

Reply:
I had to add the same data in both the PRs so that unit tests are passed during the CI run. Its in my mind to resolve these conflicts, though with every merge, I have conflicts as the same file is being written. I resolve conflicts with every merge in all the raised PRs.